app/src/main/java/com/example/keniphhomemaintenance/database/DwellingWithMaintenanceItems.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.keniphhomemaintenance.database
+
+import androidx.room.Embedded
+import androidx.room.Relation
+import com.example.keniphhomemaintenance.dwellings.Dwelling
+import com.example.keniphhomemaintenance.maintenance_screen.MaintenanceItem
+
+data class DwellingWithMaintenanceItems (
+    @Embedded val dwelling: Dwelling,
+    @Relation(
+        parentColumn = "dwellingID",
+        entityColumn = "dwellingID"
+    )
+    val maintenanceItems: List<MaintenanceItem>
```

Comment:
MaintenanceItems are nested, in a sense, to the Dwellings table. Each maintenance Item now has the dwellingID as a field